### PR TITLE
Mpp prod move

### PR DESCRIPTION
**Pull request by mfocko**
Rebased on top of @majamassarini PR, so it's easy to retry next time.

### CODE REVIEW

File: openshift/nginx.yml.j2 [hunk 1]
```diff
@@ -135,3 +139,21 @@ spec:
...
+  name: nginx
+spec:
+  tags:
+    - name: {{ deployment }}
+      from:
+        kind: DockerImage
+        name: ghcr.io/nginxinc/nginx-unprivileged
+      importPolicy:
+        # Periodically query registry to synchronize tag and image metadata.
+        # DOES NOT WORK on Openshift Online.
```
Comment (jpopelka) [inline openshift/nginx.yml.j2:155]:
```suggestion
```

File: openshift/pushgateway.yml.j2 [hunk 2]
```diff
@@ -54,3 +58,21 @@ spec:
...
+  name: pushgateway
+spec:
+  tags:
+    - name: {{ deployment }}
+      from:
+        kind: DockerImage
+        name: ghcr.io/zapier/prom-aggregation-gateway:v0.7.0
+      importPolicy:
+        # Periodically query registry to synchronize tag and image metadata.
+        # DOES NOT WORK on Openshift Online.
```
Comment (jpopelka) [inline openshift/pushgateway.yml.j2:74]:
```suggestion
```

File: docs/deployment/logs.md [hunk 3]
```diff
@@ -21,33 +15,25 @@ First, you have to [get access to Splunk](https://source.redhat.com/departments/
 
 Then go to https://rhcorporate.splunkcloud.com → `Search & Reporting`
 
+You should be able to see some logs using [this query](https://rhcorporate.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22rh_paas%22%20source%3D%22%2Fvar%2Flog%2Fcontainers%2Fpackit-worker*.log"):
+
+    index="rh_paas" source="/var/log/containers/packit-worker*.log"
+
+If the above query doesn't return any results, [request access](https://source.redhat.com/departments/it/splunk/splunk_wiki/faq#jive_content_id_How_do_I_request_access_to_additional_data_sets_in_Splunk) to `rh_paas` index.
```
Comment (nforro) [inline docs/deployment/logs.md:22]: I just went through this and there is no `Access to Additional Datasets` choice in the Service Now request form. I used `Update Permissions` as Request Type and asked for access to `rh_paas` index in additional details.

Comment (mfocko) [reply docs/deployment/logs.md:22]: I'll add it to the docs :+1: thanks

**Review by majamassarini — APPROVED** [review]
For what I can grasp this looks good to me.
Thanks a lot!

**Pull request merged**